tests/classifiers/test_ddc.py

```diff
@@ -23,42 +22,42 @@ def test_audience(self):
         def aud(identifier):
             return DDC.audience(*DDC.scrub_identifier(identifier))
 
-        eq_(child, aud("JB"))
-        eq_(child, aud("J300"))
-        eq_(child, aud("NZJ300"))
-        eq_(child, aud("E"))
-        eq_(young_adult, aud("Y300"))
-        eq_(None, aud("FIC"))
-        eq_(None, aud("Fic"))
+        assert child == aud("JB")
+        assert child == aud("J300")
+        assert child == aud("NZJ300")
+        assert child == aud("E")
+        assert young_adult == aud("Y300")
+        assert None == aud("FIC")
+        assert None == aud("Fic")
 
 
         # We could derive audience=Adult from the lack of a
         # distinguishing "J" or "E" here, but we've seen this go
         # wrong, and it's not terribly important overall, so we don't.
-        eq_(None, aud("B"))
-        eq_(None, aud("400"))
+        assert None == aud("B")
+        assert None == aud("400")
 
 
     def test_is_fiction(self):
 
         def fic(identifier):
             return DDC.is_fiction(*DDC.scrub_identifier(identifier))
 
-        eq_(True, fic("FIC"))
-        eq_(True, fic("E"))
-        eq_(True, fic(813))
+        assert True == fic("FIC")
+        assert True == fic("E")
+        assert True == fic(813)
 
-        eq_(False, fic("JB"))
-        eq_(False, fic("400"))
-        eq_(False, fic("616.9940092"))
-        eq_(False, fic(615))
-        eq_(False, fic(800))
-        eq_(False, fic(814))
+        assert False == fic("JB")
+        assert False == fic("400")
+        assert False == fic("616.9940092")
+        assert False == fic(615)
+        assert False == fic(800)
+        assert False == fic(814)
 
     def test_classification(self):
         def c(identifier):
             i, name = DDC.scrub_identifier(identifier)
-            eq_(name, None)
+            assert name == None
```

Comment:
I totally agree it would be nice to use the `is` operator here. I think there are a lot of improvements we could make to these tests to better utilize pytest. It would be great to convert a lot of the fixtures to use the pytest fixture system as well.

For this pass though the code though, since there there are a LOT of assertions, I used the [nose2pytest](https://github.com/pytest-dev/nose2pytest) tooling which made these changes, rather then updating the code by hand. 

The old [`eq_` function](https://github.com/nose-devs/nose/blob/7c26ad1e6b7d308cafa328ad34736d34028c122a/nose/tools/trivial.py#L28) we were calling was doing this same direct comparison.
```python
def eq_(a, b, msg=None):
    """Shorthand for 'assert a == b, "%r != %r" % (a, b)
    """
    if not a == b:
        raise AssertionError(msg or "%r != %r" % (a, b))
``` 

Maybe we could leave further pytestification of the code to future work?

Reply:
Oh definitely--I don't think it's a blocker at all.